src/webgpu/util/conversion.ts

```diff
@@ -111,6 +111,18 @@ export const kFloat32Format = { signed: 1, exponentBits: 8, mantissaBits: 23, bi
 /** FloatFormat defining IEEE754 16-bit float. */
 export const kFloat16Format = { signed: 1, exponentBits: 5, mantissaBits: 10, bias: 15 } as const;
 
+const workingData = new ArrayBuffer(4);
+const workingDataU32 = new Uint32Array(workingData);
+const workingDataF32 = new Float32Array(workingData);
+export function float32BitsToNumber(bits: number): number {
+  workingDataU32[0] = bits;
```

Comment:
I think the take away here is don't create temporary TypedArrayBuffer for reinterpretation.

Reply:
I measured the performance of this briefly while testing a bunch of other things. It didn't have a large effect, but it was enough that it seemed worth using.

However I tried measuring it against just now (by just moving `workingData*` inside these functions) and I wasn't able to measure a difference... ~2020ms either way. Maybe it got optimized better somehow when written this way?

Reply:
Oh, the test case I was using is no longer bottlenecked on this function. I tested a different test case (rgba32float) which is, and the results are good.
`webgpu:web_platform,copyToTexture,ImageBitmap:from_ImageData:alpha="premultiply";orientation="flipY";srcDoFlipYDuringCopy=false;dstColorFormat="rgba32float";dstPremultiplied=true`

preallocated (this PR): 1640ms
late allocated (same but workingData moved inside the function): 2130ms
array-initialized (`new Float32Array(new Uint32Array([bits]).buffer)[0]`): 2260ms

incidentally I realized one of these functions is implemented wrong, so fixing that.

Reply:
Ok, so it seems that the take away is still correct!